tests/integration/pilot/workloadentry_test.go

```diff
@@ -105,7 +104,28 @@ spec:
   workloadSelector:
     labels:
       app: z
----
+`
+			// Configure ServiceEntry
+			t.ConfigIstio().YAML(namespaceName, serviceEntryYaml).ApplyOrFail(t, apply.CleanupConditionally)
+
+			type testCase struct {
+				name    string
+				port    int
+				address string
+				network string
+			}
+			testCases := []testCase{
+				{"gateway address", ewGatewayPort, ewGatewayIP, ""},
```

Comment:
need to cover other cases too

Reply:
Like the ones in the unit test with subsets and things?